internal/common/monitors_runner.go

```diff
@@ -0,0 +1,76 @@
+package common
```

Comment:
probably should be under package and not under common.

Reply:
was not sure